Add explanation to assist in parameterized_docker_build.sh use

SaintNazaire opened the pull request:
Explains that using parameterized_docker_build.sh is dependent upon running from an appropriate developer image.

SaintNazaire committed:
Add explanation to assist in parameterized_docker_build.sh use

Explains that using parameterized_docker_build.sh is dependent upon running from an appropriate developer image.

tensorflow-jenkins commented:
Can one of the admins verify this patch?

gunan commented:
@SaintNazaire Please sign the Google CLA, and respond on this thread with the words "I signed it!" to unblock the CLA check.

gunan commented on tensorflow/tools/docker/README.md:
```diff
@@ -59,6 +59,13 @@ Building TensorFlow Docker containers should be done through the
 script. The raw Dockerfiles should not be used directly as they contain strings
 to be replaced by the script during the build.
 
```
Please add line wrappin, make all lines shorter than 80 characters.

gunan commented on tensorflow/tools/docker/README.md:
```diff
@@ -59,6 +59,13 @@ Building TensorFlow Docker containers should be done through the
 script. The raw Dockerfiles should not be used directly as they contain strings
 to be replaced by the script during the build.
 
+Attempting to run [parameterized_docker_build.sh] from a binary docker image such as for example tensorflow/tensorflow:latest will not work. One needs to execute the script from a developer docker image since by contrast with a binary docker image it contains not only the compiled solution but also the tensorflow source code. Please select the appropriate developer docker image of tensorflow at t ...
```
[parameterized_docker_build.sh] should be followed by (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/parameterized_docker_build.sh) to properly have the hyperlink

gunan replied:
Please wrap tensorflow/tensorflow:latest with "`"

gunan replied:
Instead of right tag, let's just type <tag>.
Also wrap the tensorflow/tensorflow:[right tag] inside a code block with "`"

gunan commented on tensorflow/tools/docker/README.md:
```diff
@@ -59,6 +59,13 @@ Building TensorFlow Docker containers should be done through the
+Attempting to run [parameterized_docker_build.sh] from a binary docker image such as for example tensorflow/tensorflow:latest will not work. One needs to execute the script from a developer docker image since by contrast with a binary docker image it contains not only the compiled solution but also the tensorflow source code. Please select the appropriate developer docker image of tensorflow at t ...
+
+The smallest command line to generate a docker image will then be:
+docker run -it tensorflow/tensorflow:right tag
```
Wrap the whole line inside "```" block
also replace "right tag" with <tag>

SaintNazaire replied:
Replaced "right tag" with "right_tag" is that what you meant?

gunan replied:
Looks like I meant to say more, but I completely forgot what I wanted to say here :)

gunan commented on tensorflow/tools/docker/README.md:
```diff
@@ -59,6 +59,13 @@ Building TensorFlow Docker containers should be done through the
+The smallest command line to generate a docker image will then be:
+docker run -it tensorflow/tensorflow:right tag
+
+Port mapping at 8888 -p 8888:8888 is nice to have.
```
Replace with a better explanation:
"If you would like to start a jupyter notebook on your docker container, make sure to map the port `8888` of your docker container by adding `-p 8888:8888` to the above command."

SaintNazaire committed:
Updated to comply with Gunan comments 

- Please add line wrappin, make all lines shorter than 80 characters. [DONE]
- [parameterized_docker_build.sh] should be followed by (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/parameterized_docker_build.sh) to properly have the hyperlink [DONE]
- Please wrap tensorflow/tensorflow:latest with "`" [DONE]
- Instead of right tag, let's just type . [DONE]
- Also wrap the tensorflow/tensorflow:[right tag] inside a code block with "`" [DONE]
- Wrap the whole line inside "```" block [DONE]
- also replace "right tag" with [replaced by "right_tag" is that what you meant or you prefer "."?]
- Replace with a better explanation: "If you would like to start a jupyter notebook on your docker container, make sure to map the port 8888 of your docker container by adding -p 8888:8888 to the above command." [DONE]

gunan commented:
Jenkins, test this please.
